documentation/content/Contributing/git-commit-guidelines.md

```diff
@@ -1,111 +1,150 @@
 ---
 
 [TOC]
 
-Elegant release process is [fully automated]({filename}./automated-release.md). It only works if all commit messages adhere to the set rules.
+The Elegant release process is [fully automated]({filename}./automated-release.md). To make
+this work, all commit message must adhere to a given set of rules.
+
+## Why Have Commit Message Rules?
 
-Why?
+From the [semantic-release project](https://github.com/semantic-release/semantic-release#commit-message-format):
 
 > semantic-release uses the commit messages to determine the type of changes in the codebase. Following formalized conventions for commit messages, semantic-release automatically determines the next semantic version number, generates a changelog and publishes the release.
 
-Basically, semantic-release goes through the commit messages, parses them and on its bases makes the decisions of publishing new release and new version number.
+What does that mean? It means that semantic-release parses the commit messages to arrive
+at a unbiased version number for a new release, and then creates that new release. This
+allows for a new release to be put together with little human involvement.
+
+## What are those rules?
+
+The Elegant development team chose to use the
+[Angular Commit Message Conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines)
+as the baseline for the team's commit message conventions.
 
-What are those rules?
+Our conventions are largely derived from the Angular team's guidelines
+[as documented here](https://gist.github.com/stephenparish/9941e89d80e2bc58a153).
 
-Elegant development team chose to use [Angular Commit Message Conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines).
+### Is There Something to Help Me With The Rules?
 
-Following portion is largely derived from their [guidelines](https://gist.github.com/stephenparish/9941e89d80e2bc58a153).
+Reading, understanding, and then getting used to following guidelines may take time. Even
+then, if you are in a hurry, you can sometimes forget what the rules are. It's only human.
 
-!!! Tip "Use Commitizen"
+Our team has found life easier since we started using
+[Commitizen for Git commits]({filename}./commitizen.md).
+Our project includes configuration for Commitizen that automatically formats each commit
+message to conform to our guidelines by walking you through a series of prompts. When you
+finish those prompts, a new commit messages is authored for you with the information from those
+prompts, following all of the rules in the following sections on the Commit Message Format.
 
-    Reading, understanding and then getting used to following guidelines may take sometime.
+!!! tip
 
-    Make your life easier and [use Commitizen for Git commits]({filename}./commitizen.md).
-    It automatically formats the commit message to conform to our guidelines.
+    Use Commitizen.  It does make life easier.
```

Comment:
Please re-add link to commit-zen help article. and "  It automatically formats the commit message to conform to our guidelines."

Reply:
good catch.  done.